.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-- "0.10"
+- 6.3.0
```

Comment:
As of October 18th node LTS will be v6. We can switch back to Node v4 till then if you'd prefer, or ?